testnet/src/neon_node.rs

```diff
@@ -28,15 +28,18 @@ use stacks::util::vrf::VRFPublicKey;
 use stacks::util::get_epoch_time_secs;
 use stacks::util::strings::UrlString;
 use stacks::util::hash::Hash160;
+use stacks::util::hash::Sha256Sum;
+use stacks::util::secp256k1::Secp256k1PrivateKey;
 use stacks::net::NetworkResult;
+use stacks::net::PeerAddress;
 use std::sync::mpsc::{sync_channel, TrySendError, SyncSender, Receiver};
 use crate::burnchains::bitcoin_regtest_controller::BitcoinRegtestController;
 use crate::ChainTip;
 
 use stacks::burnchains::BurnchainSigner;
 
 pub const TESTNET_CHAIN_ID: u32 = 0x00000000;
```

Comment:
This `TESTNET_CHAIN_ID` probably need to change?

Reply:
Yes, it should.  To `0x80000000`

Reply:
Fixed